app/routes/missions.km3net/route.mdx

```diff
@@ -0,0 +1,102 @@
+The areas containing 50%, 68% and 90% of the events are used to define contours that are reported in the alert.
+
+The 50%, 68% and 90% contours are used to produce a **HEALPix skymap**, distributed as a fits file. The value of each pixel of this map is the probability per square degrees. 
+
+
+Finally, after having identified an interesting neutrino candidate and computed its errors contours, we run a **search for astrophysical counterparts**. We look for Blazars, GRBs, TDEs and 𝜇Quasars in the 90% error region of our alert candidate, cross-checking the results of 4FGL, TeVCat, XRB, Rosat and RFC catalogs. Each source is ranked based on its distance to the alert, the flux reported in the catalog and eventual timing information.
+If this counterpart search is not done, the flag "potential_counterpart" is set at "No Search", if it is done but returns no result, the flag is set at "false". If counterparts are founds, that flag is set at "true.
+In this latter case, the list of sources is reported in the alert under "counterpart_info".
+
+### GCN notice types
+Notices are distributed both as JSON and VOEvent.
+
+#### JSON-Serialized GCN Notices Types in GCN Kafka
+
+KM3NeT [GCN schema](/schema/stable/gcn/notices/km3net/alert.schema.json) and example JSON messages ([High-Energy detection](/schema/stable/gcn/notices/km3net/alert.example.json), [Multiplet detection](/schema/stable/gcn/notices/km3net/alert.multiplet.example.json)) files are available. See the [Schema Browser](/docs/schema/stable/gcn/notices/km3net/alert.schema.json) for more information on the properties defined in the schema. Detailed description and examples of KM3NeT Notices are available in the [GCN Schema GitHub project](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/km3net).
+
+<div className="overflow-table">
+
+| Type                 | Contents                                                                                           | Latency |
+| -------------------- | -------------------------------------------------------------------------------------------------- | ------- |
+| `gcn.notices.km3net` | Localization, time of arrival, FAR, Neutrino details (local direction, topology, observed energy)  | ~1 min  |
+
+</div>
+
+
+#### VOEvent XML-serialized GCN notice types
+
+ <div className="overflow-table">
+
+| Type                         | Contents                                                                                           | Latency |
+| ---------------------------- | -------------------------------------------------------------------------------------------------- | ------- |
+| `gcn.notices.km3net.voevent` | Localization, time of arrival, FAR, Neutrino details (local direction, topology, observed energy)  | ~1 min  |
+
+ </div>
```

Comment:
Please add examples of voevent notices.  See how SVOM did that on their mission page: https://gcn.nasa.gov/missions/svom